python/test/gluon/test_core.py

```diff
@@ -185,23 +180,28 @@ def kernel(a_ptr, b_ptr, c_ptr, stride_am, stride_ak,  #
         offs_c = offs_cm[:, None] * stride_cm + offs_cn[None, :] * stride_cn
         ttgl.amd.cdna3.buffer_store(stored_value=c, ptr=c_ptr, offsets=offs_c)
 
-    if not is_hip_cdna4():
-        return []
+    if not is_hip_cdna4() and not is_hip_cdna3():
+        pytest.skip()
```

Comment:
Can you provide proper skip messages? Something like "AMD CDNA3/CDNA4 specific test"